tests/test_models.py

```diff
@@ -34,10 +34,19 @@ def test_debug_runs(self):
 
     @pytest.mark.filterwarnings("ignore:.*Zigzag_1S14.*:UserWarning")
     def test_simulation_runs(self):
-        # With a recombination_map of None, we simulate a coalescent without
-        # recombination in msprime, with mutation rate equal to rate from model.
+        # When model has no specific recombination rate,
+        #   we simulate coalescent without recombination.
+        # Otherwise, we use the rate from the model.
+        # We use mutation rate equal to rate from model
+        #  (or species when the model does not have one.
```

Comment:
```suggestion
        # For the purposes of testing the model, we use the
        # model's mutation and recombination rate if it has them;
        # otherwise, we use zero recombination and the species
        # mutation rate.
```